pkgs/tools/misc/rshim-user-space/default.nix

```diff
@@ -24,12 +27,15 @@ stdenv.mkDerivation rec {
     autoconf
     automake
     pkg-config
+    makeBinaryWrapper
   ];
 
   buildInputs = [
     pciutils
     libusb1
     fuse
+    busybox
+    pv
```

Comment:
This shouldn't be needed if you wrap them explicitly.

Reply:
removed it